bannou/database/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ..context import BaseMeta
+
+from .tags import Tag
```

Comment:
```suggestion
from bannou.database.context import BaseMeta

from bannou.database.models.tags import Tag
```

Relative imports can be really hard to follow and maintain

Reply:
Gotcha, will keep to this practice